### ISSUE TITLE
[CAIA Intake] Form 21-4138

### ISSUE THREAD

**Issue by tbaker1026** [department-of-veterans-affairs/va.gov-team#75607]
# Content, accessibility, information architecture (CAIA) new initiative collaboration request

Use this ticket to start collaboration on a new initiative with the sitewide content, accessibility, and information architecture (CAIA) team.

**Note:** If you’re already partnering with us on an initiative, you don't need to fill out this request form. If you don’t have access to your initiative’s CAIA epic, post a message in the [#sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-acessibility-ia) slack channel and tag `@Terry Nichols` and `@Lily Strelich`.

## About your team

- **Team name**: Veteran Facing Forms
- **OCTO product owner:** Matthew Dingee
- **Product name:**  VA Form 21-4138 Statement in Support of a Claim
- **Product manager:**  Adam Oyenuga 
- **Slack channel:** veteran-facing-forms
- **Dedicated content writer on your team (if you have one):** `Insert name and/or GH handle`
- **Dedicated a11y specialist on your team (if you have one):** `Insert name and/or GH handle`
 
### Which of these descriptions best fits the work we’ll partner on?
Select all that apply.
- [x] Digitizing a new form
- [ ] Creating a new digital tool
- [ ] Updating an existing form or tool
- [ ] Translating a form or tool into Spanish
- [ ] Adding new unauthenticated content to VA.gov
- [ ] Updating existing unauthenticated content on VA.gov
- [ ] Something else:  `insert description`

### What's the nature of your initiative and desired outcomes?

`Tell us briefly about your work and the outcomes you’re aiming to achieve.`

## Collaboration timeframe

**Note:** We work on nearly every OCTO product and manage all unauthenticated content on VA.gov, so we will need to prioritize intake requests based on overall workload and VA and OCTO priorities.

### Is this work tied to a Congressional mandate, change in law or policy, or upcoming event with a specific deadline?
- [ ] Yes
- [x] No
- [ ] I’m not sure

### Where are you at in your timeline?
Tell us briefly about what you're working on now (such as initial discovery, wireframing, or usability research planning) and add any known dates for upcoming milestones or deadlines.

`Enter your description here`

### Which phases of the collaboration cycle have you completed?
Select all that apply.
- [ ] Design intent 
- [x] Midpoint review 
- [ ] Staging 
- [ ] None. This initiative isn’t going through the collaboration cycle.

### Collaboration cycle ticket 

If you’re going through the collaboration cycle, provide your ticket number and link:
- **Ticket Number:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/75601
- 
## Supporting artifacts
Provide links to any supporting artifacts that can help us better understand your initiative and begin collaboration. Include artifacts like your product outline, user flows, mockups and prototypes, or any draft content.
- [Supporting artifact 21-4138](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/main/Forms/21-4138/Supporting%20Artifacts.md) 
- 

## Next steps
- [ ] Post a link to this issue in the [#sitewide-content-sitewide-content-accessibility-ia](https://dsva.slack.com/channels/sitewide-content-accessibility-ia) slack channel and tag `@Terry Nichols`. We’ll reply within 24 hours to acknowledge receipt. We'll then review the artifacts in more detail and work with you to determine timing for collaboration. Depending on the work, we may schedule a kickoff meeting to better understand how we'll partner together.
- [ ] Review the remaining next steps in the **VFS team tasks** section here.

### VFS team tasks
- [ ] **If you also need engineering support from the public websites team**, fill out their intake request form as well.<br> [Open a public websites intake request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E)
- [ ] **If you're requesting a page/URL redirect, a URL change, or a vanity URL**, submit an additional request for this work. <br> [Open a redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=jennymayoco%2C+kristinoletmuskat%2C+strelichl%2C+FranECross&labels=sitewide+CAIA%2C+Sitewide+IA%2C+Public+Websites%2C+VA.gov+frontend%2C+Redirect+request&projects=&template=redirect-request.md&title=Redirect+Request)
- [ ] **If you're creating an experimental design**, also contact the design system team. <br> [Read more about experimental designs](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) <br> [Suggest an addition or update to the design system team](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)
- [ ] **If you're requesting a11y support for usability research**, submit a research support request next. <br> [Open an a11y research support request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name)
- [ ] **If you are requesting content support**, please share your content source of truth (preferably as a SharePoint document).

### CAIA operations
- [x] Initial Review - @coforma-terry and @strelichl

**Comment by megzehn** [comment]
Moving this one to in progress, because the team is ready for us to start providing content feedback. 

I've opened a PR and [started an initial review](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/blob/megzehn-content-review-4138/Forms/21-4138/source%20of%20truth.md). I plan to return to this when I get back on April 2.

**Comment by megzehn** [comment]
Just flagging that content work on this is ongoing.

A few updates:

- @rachelshearerux confirmed yesterday that we're going to be focusing the digital experience on Veterans and claimants only (not including a path for POAs). 

- After reviewing midpoint feedback, I'm looking at other ways of approaching the content in the intro page (either a separate R&S article or revising the supporting forms page).

**Comment by AdamOye** [comment]
closed by mistake. Reopening

**Comment by megzehn** [comment]
Update: I finished reviewing 21-4138 and sent the[ PR with SOT updates](https://github.com/department-of-veterans-affairs/VA.gov-team-forms/pull/1229/files) to @rachelshearerux!

**Comment by kristinoletmuskat** [comment]
Hey @tbaker1026 -- is this ticket still open? Are you expecting any additional work from us on this?

**Comment by tbaker1026** [comment]
@kristinoletmuskat No we are good - I can close it.

**Comment by tbaker1026** [comment]
@mnorthuis - We will need content for the Flipper widget for the /supporting-forms-for-claims/statement-to-support-claim-form-21-4138 page.

**Comment by tbaker1026** [comment]
Sample copy from another form for the flipper UI - https://dsva.slack.com/canvas/D05CJN00LCQ

**Comment by mnorthuis** [comment]
Dropping info from IA tracker ticket here for single SOT on remaining work. 

> ### Decision Review
> Background: User research led the designs to be less complicated. Instead of trying to guide users to what type of decision review with code, we are just going to take them directly to the decision-reviews option for the user to read more about it.
> 
> **CURRENT IA Spec** Remove all of the below What to know before you request a decision review | Asks about the claim's decision date| `/decision-review` There's a better way to ask for a decision review | The decision is older than a year, so we suggest filing a supplemental claim | `/decision-review-over-one-year` Which description is true? | Select disagreement reason | `/decision-review-type` There's a better way for you to ask for a decision review | They have new evidence, so we suggest filing a Supplemental Claim | `/decision-review-supplemental-claim` There's a better way for you to ask for a decision review | Suggests requesting a Higher-Level Review | `/decision-review-higher-level-review` There's a better way for you to ask for a decision review | Suggests requesting a Board Appeal | `/decision-review-board-appeal`
> 
> **Change to be:** /supporting-forms-for-claims/statement-to-support-claim-form-21-4138/decision-reviews
> 
> ### Priority Processing
> Background: Priority processing, before we were trying to get some information up front about why they were requesting priority processing, now we are just going to take them directly to the form.
> 
> **CURRENT IA Spec: Remove** What to know before you request priority process | Priority processing introduction | `/priority-processing` Which of these statements best describes your living situation? | Select a housing situation | `/priority-processing-living-situation` Other housing risks | Other housing risks | `/priority-processing-housing-risks` Are there any of these descriptions true for you? | Other hardships | `/priority-processing-hardships` You may not qualify for priority processing | Not qualified for priority processing, so we suggest submitting a 4138 statement | `/priority-processing-statement` There's a better way to request priority processing | Suggests requestion priority processing | `/priority-processing-request`
> 
> **Change to be:** /supporting-forms-for-claims/statement-to-support-claim-form-21-4138/priority-processing-request
> 
> ### Personal records request
> **Current IA spec:** There's a better way to request your personal records | Suggests requesting personal records | `/records-request`
> 
> **Change to be:** /supporting-forms-for-claims/statement-to-support-claim-form-21-4138/priority-processing-request
> 
> ### Submit new evidence
> **Current IA spec:** There's a better way to submit new evidence | Suggests claim status tool | `/submit-evidence`
> 
> **Change to be:** /supporting-forms-for-claims/statement-to-support-claim-form-21-4138/submit-new-evidence

**Comment by RLHecht** [comment]
@tbaker1026 The link you posted isn't working for me. Could you copy and paste the sample text here? Thank you!

**Comment by tbaker1026** [comment]
@RLHecht Here is another forms sample content and screenshot
Widget copy for form 20-10207 

[For people who can access the form]

 
You can submit this form online or by mail.

[blue secondary action link] [Submit a request online for priority processing for an existing claim](http://www.va.gov/supporting-forms-for-claims/request-priority-processing-form-20-10207)
 

[For people who can’t access the form]

 
You can submit this form by mail.

<img width="707" alt="Image" src="https://github.com/user-attachments/assets/cf21284e-99ab-45a0-8623-3775c9ad3138" />

**Comment by mnorthuis** [comment]
@tbaker1026 Reviewing documentation and this form in staging does not reflect the IA approved URL.  Can you tell me what required the change from what was provided?

URL provided in IA spec:   www.va.gov/supporting-forms-for-claims/submit-statement-form-21-4138/
Current URL in staging: staging.va.gov/supporting-forms-for-claims/statement-to-support-claim-form-21-4138/

**Comment by tbaker1026** [comment]
@mnorthuis - I will get it updated to the approved IA spec. @RLHecht I will let you know when the new URL is in place to update the supporting forms page.
